readme.md

```diff
@@ -0,0 +1,34 @@
+# Emojo
+Lambda function to be used as a slack app to add uploaded emoji to github repo to then be deployed to slack through CI.
+
+### Config
+All config is done through environment variables. These can be set in a `.env` file in the root of the project. See `.env.example`
+
+#### `GITHUB_REPO`
+Name of the repo you wish to save your emoji too
+
+#### `GITHUB_REPO_DIR`
+A sub-directory in the repo where the images themselves will be added.
+
+#### `GITHUB_REPO_BRANCH`
+Custom branch to commit the uploaded emoji to
+
+### Deploy
+Emojo is built using [`serverless`][1] so once that is configured you can run:
```

Comment:
Don't need to set it up, it's a dev dep. Unless you mean your aws creds? SLS will warn you about those if you run it without them and help you set them up.

Reply:
Yea that's what I mean, there is still some setup required for serverless

Reply:
Don't need to describe it though probably i mean as as soon as you run `yarn deploy` it'll either run, or ask you for your aws creds. the only actual step is to run `yarn deploy`